docs/development/dependencies/requirements.mdx

```diff
@@ -13,24 +14,156 @@ description: "Use 3rd party Python packages in your project and pipelines."
 
 ## Add Python packages to `requirements.txt`
 
+### Location
+
+At the root of your Mage project directory, you'll find a file named `requirements.txt`. 
+This file is used to specify all the Python packages that your project depends on.
+
+### Path
+
 At the root of your project directory, there is a file named `requirements.txt`. Open this file
 and add the Python package you want installed and available throughout your entire project.
 
 For example, if your project is named `demo_project` then the file will be located at
 `demo_project/requirements.txt`.
 
-## Example
+### Adding packages
 
-`[project_name]/requirements.txt`
+To add a package, simply open the `requirements.txt` file and list each package on a new line. You can also specify version requirements if needed.
+
+### Example `requirements.txt`:
 
 ```
-matplotlib
+matplotlib==3.5.2
 tqdm
 ```
+In this example the `requirements.txt` file will install:
+
+- matplotlib version 3.5.2
+- tqdm
 
 ## Automatic installation
 
-To automatically install the packages in `requirements.txt`, you'll need to make sure the
-environment variable `USER_CODE_PATH` is set to the value of your project path (e.g. `/home/src/project_name`).
+When installing Mage using Docker, Mage provides two main methods for automatically installing third-party packages at runtime.
+These methods utilize the USER_CODE_PATH environment variable, which defines your project path (e.g. `/home/src/project_name`), 
```

Comment:
@colefreeman Wrap "USER_CODE_PATH" with backticks for consistency with the rest of the doc.

Reply:
resolved and is working in mintlify dev env